platform/docs/docs/platform/extensions/modules/panel.md

```diff
@@ -64,7 +68,17 @@ providing the component with its.
 
 New: You can easily add multiple panels to the left/right side of the viewer
 using the mode configuration. As seen below, the `leftPanels` and `rightPanels`
-accept an `Array` of the `IDs`.
+accept an `Array` of the `IDs`. The default state of either/both the left and
+right side panels may also be specified. The default state includes whether the side
+panel should be closed initially and if closed initially, whether the side panel
+should open as soon as one of its child panels requests to activate via a
```

Comment:
child panels => child panels tabs? or just tabs?

Reply:
Well this whole md file does NOT mention tab(s) a single time, so I am not going to introduce that now. I think child panels is just fine.